doc/extensions/zephyr/application.py

```diff
@@ -121,6 +126,7 @@ def run(self):
         tool = self.options.get('tool', 'cmake').lower()
         app = self.options.get('app', None)
         zephyr_app = self.options.get('zephyr-app', None)
+        cd_into = 'cd-into' in self.options
```

Comment:
If it's a flag, maybe you want to make it a `directives.flag` above (like `'compact'`) to keep the code self-documenting?

Reply:
yep, good point